packages/hint-no-vulnerable-javascript-libraries/src/_locales/en/messages.json

```diff
@@ -7,6 +7,10 @@
         "description": "Report message when the execution of the script fails",
         "message": "Error executing script: '$1'"
     },
+    "learnMore": {
+        "description": "Snyk documentation link text",
+        "message": "Learn more in Snyk"
```

Comment:
I think we'll have to interpolate some data into this message to avoid having 3+ "Learn more in Snyk" links in a row. How about this?
```suggestion
        "message": "Learn more about vulnerability $1 ($2) at Snyk"
```

E.g.
```
Learn more about vulnerability SNYK-JS-ANGULAR-572020 (high) at Snyk
```

Reply:
Sounds good.

Reply:
done!